homeassistant/components/apple_tv/config_flow.py

```diff
@@ -157,7 +160,6 @@ async def async_step_reauth(
             "type": "Apple TV",
         }
         self.scan_filter = self.unique_id
-        self.context["identifier"] = self.unique_id
```

Comment:
This was set here and in `async_found_zeroconf_device`, but didn't seem to be accessed anywhere.

Reply:
I think its legacy cruft